models/post_struct.go

```diff
@@ -47,7 +47,7 @@ type PostListingChildData struct {
 	ThumbnailWidth             float64            `json:"thumbnail_width"`
 	AuthorFlairTemplateId      string             `json:"author_flair_template_id"`
 	IsOriginalContent          bool               `json:"is_original_content"`
-	UserReports                []string           `json:"user_reports"`
+	UserReports                []UserReport       `json:"user_reports"`
```

Comment:
This is a little bit of a breaking change but with the update requested here, I think it's okay